workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGen/EntryPoint.cs

```diff
@@ -43,27 +49,88 @@ public CodeGenerator(CodeGeneratorOptions options, IFileSystem fileSystem)
         {
             this.options = options;
             this.fileSystem = fileSystem;
+
+            ConfigureLogger();
+            logger = LogManager.GetCurrentClassLogger();
+        }
+
+        private static void ConfigureLogger()
+        {
+            var codeGeneratorOptions =  CodeGeneratorOptions.Instance;
+
+            var config = new NLog.Config.LoggingConfiguration();
+
+            var jsonLayout = new JsonLayout
+            {
+                Attributes =
+                {
+                    new JsonAttribute("time", "${longdate}"),
+                    new JsonAttribute("level", "${level:uppercase=true}"),
+                    new JsonAttribute("logger", "${logger}"),
+                    new JsonAttribute("message", "${message}"),
+                    new JsonAttribute("exception", "${exception:format=ToString}")
```

Comment:
Will this unwrap inner exceptions too? Or will it only show top-level ones?

Reply:
This will show the inner exceptions too: https://github.com/NLog/NLog/wiki/Exception-layout-renderer#log-full-but-without-data